### PR TITLE
Add a note calling out benefits of using Github App to make it more v…

### PR DESCRIPTION
**Pull request by nsuma8989**
Add a note calling out benefits of using Github App to make it more visible to users. Below is screenshot on how the change will look like to bring more visibility to users on benefits

Screen Shot 2021-11-16 at 12 53 31 PM

### CODE REVIEW

**Comment by karensawrey** [comment]
LGTM

**Review by karensawrey — APPROVED** [review]

File: pages/integrations/github.md.erb [hunk 1]
```diff
@@ -8,7 +8,12 @@ To complete this integration, you need admin privileges for your GitHub reposito
 
 ## Connecting Buildkite and GitHub
 
-You can now use our [GitHub App](#connect-your-buildkite-account-to-github-using-the-github-app) to connect a Buildkite organization to a GitHub organization, removing the reliance on individual user connections to report build statuses. See the [changelog announcement](https://buildkite.com/changelog/102-github-app-integration).
+You can now use our [GitHub App](#connect-your-buildkite-account-to-github-using-the-github-app) to connect a Buildkite organization to a GitHub organization.
+
+<div class="Docs__note">
+<h3 class="Docs__note__heading">Benefits of using Github App</h3>
+<p>Using Github App removes the reliance on individual user connections to report build statuses. See the <a href="https://buildkite.com/changelog/102-github-app-integration">changelog announcement</a></p>
```
Comment (plaindocs) [inline pages/integrations/github.md.erb:15]:
```suggestion
<p>Using GitHub App removes the reliance on individual user connections to report build statuses. See the <a href="https://buildkite.com/changelog/102-github-app-integration">changelog announcement</a></p>
```

File: pages/integrations/github.md.erb [hunk 2]
```diff
@@ -8,7 +8,12 @@ To complete this integration, you need admin privileges for your GitHub reposito
 
 ## Connecting Buildkite and GitHub
 
-You can now use our [GitHub App](#connect-your-buildkite-account-to-github-using-the-github-app) to connect a Buildkite organization to a GitHub organization, removing the reliance on individual user connections to report build statuses. See the [changelog announcement](https://buildkite.com/changelog/102-github-app-integration).
+You can now use our [GitHub App](#connect-your-buildkite-account-to-github-using-the-github-app) to connect a Buildkite organization to a GitHub organization.
+
+<div class="Docs__note">
+<h3 class="Docs__note__heading">Benefits of using Github App</h3>
```
Comment (plaindocs) [inline pages/integrations/github.md.erb:14]:
```suggestion
<h3 class="Docs__note__heading">Benefits of using GitHub App</h3>
```

**Review by plaindocs — APPROVED** [review]
Looks good, just a couple of cases to fix.

File: pages/integrations/github.md.erb (hunk 1 above)
Comment (nsuma8989) [reply pages/integrations/github.md.erb]: Updated with suggested changes

File: pages/integrations/github.md.erb (hunk 2 above)
Comment (nsuma8989) [reply pages/integrations/github.md.erb]: Updated with suggested changes

**Comment by plaindocs** [comment]
Nice one!

**Pull request merged**